added note about orphaned users (#5503)
MicrosoftDocs/windows-itpro-docs: windows/deployment/usmt/usmt-hard-link-migration-store.md

3.  An administrator runs the LoadState command-line tool on each computer. The LoadState tool restores user state back on each computer.

> [!NOTE]

> > During the update of domain-joined computer, the profiles of the users which's SID cannot be resolved would not be migrated. In case of using Hard-Link Migration Store, it could cause the data lost.

Reviewer: ```suggestion
> During the update of a domain-joined computer, the profiles of users whose SID cannot be resolved will not be migrated. When using the Hard-Link Migration Store, it could cause a data loss.
```

Reviewer 2: Well done, nothing to add from my side.
